api/libs/api-events/src/api-event-kinds.enum.ts

```diff
@@ -20,6 +20,15 @@ export enum API_EVENT_KINDS {
   scenario__run__outputSaved__v1__alpha1 = 'scenario.run.outputSaved/v1alpha',
   scenario__run__outputSaveFailed__v1__alpha1 = 'scenario.run.outputSaveFailed/v1alpha',
   scenario__run__finished__v1__alpha1 = 'scenario.run.finished/v1alpha',
+  geofeatures__copy__submitted__v1__alpha1 = 'geofeatures.copy.submitted/v1alpha',
+  geofeatures__copy__failed__v1__alpha1 = 'geofeatures.copy.failed/v1alpha',
+  geofeatures__copy__finished__v1__alpha1 = 'geofeatures.copy.finished/v1alpha',
+  geofeatures__split__submitted__v1__alpha1 = 'geofeatures.split.submitted/v1alpha',
+  geofeatures__split__failed__v1__alpha1 = 'geofeatures.split.failed/v1alpha',
+  geofeatures__split__finished__v1__alpha1 = 'geofeatures.split.finished/v1alpha',
+  geofeatures__stratification__submitted__v1__alpha1 = 'geofeatures.stratification.submitted/v1alpha',
+  geofeatures__stratification__failed__v1__alpha1 = 'geofeatures.stratification.failed/v1alpha',
+  geofeatures__stratification__finished__v1__alpha1 = 'geofeatures.stratification.finished/v1alpha',
```

Comment:
Even tho those events relate to geofeatures, shouldn't they also be included for scenario jobs statuses?

Reply:
I am not sure, processing of features is not assigned to the scenario – as AFAIU some features can be already calculated, and features can be used with multiple scenarios

Reply:
@Dyostiq the point is rather to being able to tell (by pending job) that given scenario has some processing ongoing.

@hotzevzl thoughts?

Reply:
calculated features (through any of the three ops, copy, split, stratification) should only be usable within a single scenario though ("raw" features that are not the outcome of a processing op would be available within a project if they are supplied by the user, or to all the projects if they are preloaded by instance admins), and yes, API clients need to know if the scenario has feature processing operations pending.